src/Apps/Order/Components/TransactionDetailsSummaryItem.tsx

```diff
@@ -2,36 +2,35 @@ import { TransactionDetailsSummaryItem_order } from "__generated__/TransactionDe
 import React from "react"
 import { createFragmentContainer, graphql } from "react-relay"
 
-import {
-  Flex,
-  FlexProps,
-  Sans,
-  Serif,
-  Spacer,
-  StackableBorderBox,
-} from "@artsy/palette"
-
-export interface TransactionDetailsSummaryItemProps extends FlexProps {
+import { Flex, Sans, Serif, Spacer } from "@artsy/palette"
+import { StepSummaryItem, StepSummaryItemProps } from "Styleguide/Components"
+
+export interface TransactionDetailsSummaryItemProps
+  extends StepSummaryItemProps {
   order: TransactionDetailsSummaryItem_order
   offerOverride?: string | null
   useLastSubmittedOffer?: boolean
+  offerContextPrice?: "LIST_PRICE" | "LAST_OFFER"
```

Comment:
#minor but since we're adding a new param to this component, it would be nice to add a story to `TransactionDetailsSummaryItem` so we can visualize.

Reply:
I'll add a test as well.